src/containers/App/App.js

```diff
@@ -2,7 +2,12 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { IndexLink } from 'react-router';
 import { LinkContainer } from 'react-router-bootstrap';
-import Navbar from 'react-bootstrap/lib/Navbar';
```

Comment:
I'm not sure the changes to this file are relevant to the PR? Maybe you should revert these specific changes and include them in their own PR.